devtools/smartrequests.py

```diff
@@ -0,0 +1,350 @@
+"""SmartRequest helper classes and functions for new SMART/TAPO devices.
```

Comment:
Create new package, e.g., `helpers` and move this there. Avoids confusing anyone who comes around browsing the devtools dir :-)

Reply:
put `helpers` at the root level or as a sub package of devtools?